models/heirbrant2023boekbazaar/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+title: Boekbazaar Ontology
+acronym:
+issued: 2023
+modified:
+contributor:
+ - https://www.linkedin.com/in/joachim-heirbrant-532aa1233/
+ - https://dblp.org/pid/251/5991
+ - https://dblp.org/pid/95/6356
+keyword:
+ - digital platform ontology
+ - digital platform
+ - marketplace
+theme: Class H - Social Sciences
+editorialNote: Ontology created during an Ontology Engineering lecture by students who wanted to remain anonymous. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
+ontologyType:
+ - domain
+ - application
+language: en
+designedForTask:
+ - conceptual clarification
+ - information retrieval
+ - software engineering
+context:
+ - industry
```

Comment:
I would classify an ontology developed as part of a master thesis as research only.

Reply:
IMHO, the ontology was built for both purposes, as the author opened a company that has a product that uses this ontology. Considering that, should I keep only the research context?

Reply:
I see. Nevermind my comment then.